pynucastro/rates/rate.py

```diff
@@ -715,10 +528,11 @@ def _set_screening(self):
             if len(nucz) == 3:
                 self.ion_screen.append(nucz[2])
 
-        # if the rate is a reverse rate (defined as Q < 0), then we
+        # if the rate is a reverse rate, via detailed balance, then we
```

Comment:
This reverts the comments to before #363. Is that intentional?

Reply:
that's a merge conflict issue.  I'm fixing